go.mod

```diff
@@ -48,13 +48,21 @@ require (
 	github.com/ProtonMail/go-crypto v0.0.0-20230426101702-58e86b294756 // indirect
 	github.com/antlr/antlr4/runtime/Go/antlr/v4 v4.0.0-20230321174746-8dcc6526cfb1 // indirect
 	github.com/cloudflare/circl v1.3.3 // indirect
+	github.com/davidmz/go-pageant v1.0.2 // indirect
+	github.com/go-fed/httpsig v1.1.0 // indirect
+	github.com/google/pprof v0.0.0-20230111200839-76d1ae5aea2b // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
 	k8s.io/kube-openapi v0.0.0-20230501164219-8b0f38b5fd1f // indirect
 )
 
 replace k8s.io/client-go => k8s.io/client-go v0.25.4
 
 replace k8s.io/api => k8s.io/api v0.26.3
 
+replace code.gitea.io/sdk/gitea => code.gitea.io/sdk/gitea v0.15.1-0.20230509035020-970776d1c1e9
+
+replace github.com/tektoncd/pipeline => github.com/tektoncd/pipeline v0.48.0
```

Comment:
Why replace is required :thinking:

Reply:
Because go.mod id annoying, it otherwise trie to downgrade pipeline to a version which didn't have got in there ,

Reply:
I see